src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageFormDetails.tsx

```diff
@@ -114,7 +121,7 @@ const WorkPackageEditDetails: React.FC<Props> = ({
             />
           </FormControl>
         </Grid>
-        <Grid item xs={12} md={6} sx={{ mt: 1 }}>
+        <Grid item xs={12} md={4} sx={{ mt: 1 }}>
```

Comment:
change the grid items for Project Lead and Project Manager to be md={5} to take up the whole screen